integration/chaincode/extcc/main.go

```diff
@@ -0,0 +1,263 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	pb "github.com/hyperledger/fabric-protos-go/peer"
+)
+
+var cert string = `-----BEGIN CERTIFICATE-----
```

Comment:
More statically generated fixtures. We need to stop doing this and I know I've mentioned it in the past as area we are focusing on.

If we have fixtures that are quick to generate, we prefer they are generated in the test; if complicated to generate, we fall back to `go:generate` commands and/or scripts. If neither of those work, we expecte explicit documentation about how they were generated.